Makefile

```diff
@@ -21,7 +21,16 @@ deb: orig
 	dpkg-buildpackage -b -us -uc
 
 dist: orig
-	dpkg-buildpackage -S -us -uc
+	@mkdir -p dist
+	dpkg-buildpackage -S -us -uc --output-directory=dist/
+
+# build and sign (signing uses environment GPG_PASSPHRASE and KEYID)
+sign-and-dist:
```

Comment:
We could add `dist` as a required build step for this, and that would allow us just to call `sign-and-dist` in the workflow? Feels like the name implies it would do both!